components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -17,6 +17,7 @@ import {
 })
 export class CreateObjectModalComponent implements OnInit, OnChanges {
   @Input() visible = false;
+  @Input() projectNoteVisible = false;
```

Comment:
This component is used for token, team, and project creation, but only projects should display the new text, hence the new parameter.